tests/test_record_prevalence_healthburden_class.py

```diff
@@ -0,0 +1,365 @@
+import os
+from pathlib import Path
+
+from tlo import Date, Simulation
+from tlo.analysis.utils import parse_log_file
+from tlo.methods.fullmodel import fullmodel
+
+resourcefilepath = Path(os.path.dirname(__file__)) / '../resources'
+outputpath = Path("./outputs/")
+
+start_date = Date(2010, 1, 1)
+end_date = Date(2012, 1, 1)
+
+popsize = 1000
+seed = 42
+do_sim = True
+def check_dtypes(simulation):
+    df = simulation.population.props
+    orig = simulation.population.new_row
+    assert (df.dtypes == orig.dtypes).all()
+
+
+def find_closest_recording(prevalence, target_date, log_value, column_name, multiply_by_pop):
```

Comment:
I'm not sure of the usefulness of finding the closest reported prevalence value. Would a useful test perhaps be to check all registered modules are logging prevalence every month and the inverse of this (no prevalence reported if module not registered), or set the incidence to 0 for one disease and check logger not reporting anything above 0, assert prevalence values for 2010 within reasonable range, e.g. test for extreme or unlikely values.

Reply:
We had a long discussion about the test yesterday on our call Part II. We decided to include a dummy disease with which to compare prevalences (this will be in the new test file), and to see if the prevalence of what it reports matches with what has been reported it its own logging file. 

I suppose by doing it the way that I was doing it, I was trying to see if the calculations themselves were working, as well as the general mechanics of logging. But do you think such a test is unnecessary? And that by showing e.g. with a dummy module and/or what you have suggested above, it would suffice?